gui.py

```diff
@@ -268,15 +268,31 @@ def populateDescription(self, item):
                     self.attributes[item.text()]["Default"]
         self.label.setText(info)
 
-def main():
+if __name__ == "__main__":
     gui_application = QApplication() #create new application
-    main_window = MainWindow() #create new instance of main window
+    catalog = json.load(open('result_new.json'))
+    main_window = MainWindow(catalog) #create new instance of main window
     config.mainWindow = main_window
     main_window.show() #make instance visible
     main_window.raise_() #raise instance to top of window stack
     gui_application.exec_() #monitor application for events
     gui_application.quit()
 
 
-if __name__ == "__main__":
-    main()
+if __name__ == "__m5_main__":
+    import sys
+    sys.path.append('/home/parallels/gem5/configs')
```

Comment:
This shouldn't be a hardcoded path. Feel free to make the assumption that this is run in the gem5 directory or that gem5 is in the directory above (either is fine)

Reply:
Okay, yea I meant to change it but it slipped out of my head. I'll get to it soon!